lib/engine/game/g_1871/meta.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require_relative '../meta'
+
+module Engine
+  module Game
+    module G1871
+      module Meta
+        include Game::Meta
+
+        DEV_STAGE = :prealpha
+
+        GAME_TITLE = 'The Old Prince'.freeze
+        GAME_SUBTITLE = '1871'.freeze
+        GAME_FULL_TITLE = 'The Old Prince: 1871'.freeze
+        GAME_DROPDOWN_TITLE = 'The Old Prince: 1871'.freeze
+
+        GAME_DESIGNER = 'Lucas Boyd'.freeze
+        GAME_IMPLEMENTER = 'Christopher Giroir'.freeze
```

Comment:
What is this used for?

Reply:
I filled it out since it was in the base meta file: https://github.com/tobymao/18xx/blob/master/lib/engine/game/meta.rb#L21

Looks like it's displayed in with the game info:
https://github.com/tobymao/18xx/blob/master/assets/app/view/game/game_meta.rb#L14

Reply:
I added it for 1828 because the author wanted is to use the licensing model instead of providing permission. The 18xx.games version is not associated with JC and this was the way Toby and I came up with to associate it with someone (me).

I'm not sure if any other games use it.

Reply:
OK, thanks.